glommio/src/executor.rs

```diff
@@ -915,6 +896,24 @@ impl LocalExecutor {
         }
     }
 
+    // this happens if we spawn() into the executor before we run().
+    // I am considering making this illegal, but in the meantime...
+    fn activate_pre_spawned_queues(&self) {
```

Comment:
If we can no longer `spawn` before `run`, why is this `fn` here?

Reply:
should have been deleted. It was added to handle the case where we spawn before run and I forgot to delete. Thanks!

Reply:
I am now wondering if I haven't simply uploaded the wrong version entirely? The spawn functions are still public =(